modules/govuk/manifests/apps/link_checker_api.pp

```diff
@@ -49,6 +49,10 @@
 #   start.
 #   Default: undef
 #
+# [*rate_limit_bypass_token*]
```

Comment:
Can we rename this to `rate_limit_token` for consistency with the other apps?

Reply:
Sure @thomasleese, have updated.